src/sandesh/Assignment_25/PrintFirst5PrimeNumWithinRange.java

```diff
@@ -0,0 +1,64 @@
+/*
+Program 1:  Write a method which will return first 5 prime numbers from given range. Main method should print the output.
+Input : Start Index -> 10
+        End Index -> 50
+Output : 11 13 17 19 23
+*/
+
+package sandesh.Assignment_25;
+
+import java.util.Scanner;
+
+public class PrintFirst5PrimeNumWithinRange {
+
+	int[] prime5Array = new int[5];
+	int countOfPrimeNumbers;
+	boolean checkIfPrime(int numInput) {
+		boolean isPrime = true;
+		for(int innerIndex=2; innerIndex<numInput/2 + 1; innerIndex++) {
```

Comment:
you can do <= as well

Reply:
Sure Harsh, will work on this in next assignment. Thank you.